sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -80,6 +82,7 @@ struct sof_sdw_dai_info {
 	int (*exit)(struct snd_soc_card *card, struct snd_soc_dai_link *dai_link);
 	int (*rtd_init)(struct snd_soc_pcm_runtime *rtd);
 	bool rtd_init_done; /* Indicate that the rtd_init callback is done */
+	unsigned long quirk;
```

Comment:
Sorry I am not following the explanations.

The commit message reads  "Add a quirk that defines if a system uses the codec speaker"

But here the quirk is not system-specific, it's part of the DAI definition....

Reply:
I will redo the commit message to try to make it more clear. The idea here is you can mark a DAI with a quirk, then that DAI will only be included on systems that have that quirk.